src/kemal/static_file_handler.cr

```diff
@@ -36,16 +36,27 @@ module Kemal
                     end
 
       file_path = File.join(@public_dir, expanded_path)
-      is_dir = Dir.exists? file_path
+      is_dir = Dir.exists?(file_path)
```

Comment:
I added parens as this seemed to be how most other things are done, plus was done further down the file.